rct_optimizations/include/rct_optimizations/serialization/problems.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include <rct_optimizations/extrinsic_hand_eye.h>
+#include <rct_optimizations/serialization/types.h>
+
+namespace YAML
+{
+template<>
+struct convert<rct_optimizations::ExtrinsicHandEyeProblem2D3D>
+{
+  using T = rct_optimizations::ExtrinsicHandEyeProblem2D3D;
+
+  static Node encode(const T &rhs)
+  {
+    Node node;
+
+    node["intr"] = rhs.intr;
+    node["camera_mount_to_camera_guess"] = rhs.camera_mount_to_camera_guess;
+    node["target_mount_to_target_guess"] = rhs.target_mount_to_target_guess;
+    node["observations"] = rhs.observations;
+
```

Comment:
There are two non-constant string labels label_target_mount_to_target and label_camera_mount_to_camera that may have been intended to be constant values, but are neither read nor written.

Reply:
Those labels are used for printing covariance information, so I didn't think it was necessary to include them in the serialization. I can add them for the sake of completeness though.

Those labels are non-const so that the user can change them if they want to; that was an intentional design choice I believe.